tests/bluetooth/bsim_bt/bsim_test_l2cap/src/common.c

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "common.h"
+
+#define LOG_MODULE_NAME common
+
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(LOG_MODULE_NAME, LOG_LEVEL_DBG);
+
+extern enum bst_result_t bst_result;
+
+void test_init(void)
+{
+	bst_result = In_progress;
+	bst_ticker_set_next_tick_absolute(WAIT_TIME);
+}
+
+void test_tick(bs_time_t HW_device_time)
+{
+	if (bst_result != Passed) {
+		FAIL("test failed (not passed after %i seconds)\n", WAIT_TIME);
+	}
+}
+
+/* Call in init functions*/
+void device_sync_init(uint device_nbr)
+{
+	uint peer = 0;
+
+	if (device_nbr == 0) {
+		peer = 1;
+	}
+
+	uint dev_nbrs[1] = {peer};
+	uint channel_nbrs[1] = {0};
+	uint *ch = bs_open_back_channel(device_nbr, dev_nbrs, channel_nbrs, 1);
```

Comment:
Consider putting `1` into a const variable and use that instead of the literal `1`.